src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,15 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => (
+  <article className={cn('TodoInfo', {
+    'TodoInfo--completed': todo.completed === true,
+  })}
+  >
+    <h2 className="TodoInfo__title">
+      { todo.title }
```

Comment:
take into account the previous comment about destructuring assignment (everywhere)
if you have questions - feel free to ask in the chat